src/components/notifications/PushPlus.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+    <div class="mb-3">
+        <label for="pushPlus-sendkey" class="form-label">{{ $t("SendKey") }}</label>
```

Comment:
please include all translation keys in `en.json`, if it is not there our kickass army of translators will be unable to translate `SendKey`.